src/handlers/reference-data/get-pop-names.js

```diff
@@ -0,0 +1,32 @@
+import { HTTP_STATUS } from '../../common/constants/http-status-codes.js'
+import { handleBackendResponse } from '../handle-backend-response.js'
+import { createLogger } from '../../common/helpers/logging/logger.js'
+import { validPopNames } from '../../common/constants/pop-names.js'
+
+const logger = createLogger()
+
+export const handleGetPopNames = async (_request, h) => {
+  try {
+    const response = {
+      statusCode: HTTP_STATUS.OK
+    }
+    const responseData = mapGetPopNamesResponse()
+
+    return handleBackendResponse(response, h, () => responseData)
+  } catch (error) {
+    logger.error({ err: error }, 'Error getting POP names')
+    return h
+      .response({
+        statusCode: HTTP_STATUS.INTERNAL_SERVER_ERROR,
+        error: 'Internal Server Error',
+        message: 'Failed to get POP names'
+      })
+      .code(HTTP_STATUS.INTERNAL_SERVER_ERROR)
+  }
+}
+
+export const mapGetPopNamesResponse = () =>
+  validPopNames.map((pop) => ({
+    code: pop.code,
+    name: pop.name
```

Comment:
let's take the name out of this PR. We will consider adding it in later in a separate ticket.